agent/tests/test_withdraw.py

```
@@ -8,12 +8,14 @@
 
 
 def _make_mock_identity(principal_str="controller-principal"):
+    """Create a mock identity for testing."""
     identity = MagicMock()
     identity.sender.return_value = MagicMock(__str__=lambda s: principal_str)
```

Ruff (0.15.2)

[warning] Unused lambda argument: `s`

(ARG005)